packages/dashboard/src/layout/useDashboardPanel.ts

```diff
@@ -56,16 +58,15 @@ export function useComponent<
       const { id: widgetId, type } = widget;
       const name = widget.title ?? widget.name;
       const isSupportedType =
-        (Array.isArray(variableName) && variableName.includes(type)) ||
-        type === variableName;
+        (Array.isArray(supportedTypes) && supportedTypes.includes(type)) ||
+        type === supportedTypes;
       if (!isSupportedType) {
         // Only want to listen for your custom variable types
         return;
       }
       const metadata = { id: widgetId, name, type };
       let props: DehydratedDashboardPanelProps & { fetch?: typeof fetch } = {
         localDashboardId: id,
```

Comment:
What is the "local" for in this variable name? Why not just `dashboardId`? It's just the ID of the dashboard the panel is mounted in, right? So in community the same for all panels, but in enterprise different for each code studio/dashboard. "local" seems like an unneeded qualifier here (what other types of `dashboardId` are there?)

Might also make sense in a context (but this one might be a bigger refactor than `registerComponent`). Maybe they go in the same context and you can have some access hook like `const { id: dashboardId, registerComponent } = useDashboardInfo();`. Just a thought that might simplify a few things or clear up some uses.

If we changed to a context, then probably should keep the props and deprecate if enterprise still needs the props.

Reply:
Yea there's no real point to having the `local` bit. I think there was something where we needed to create an ID for a dashboard in advance when we hadn't saved that dashboard yet, but no reason to name it `local`